idd/Energy+.idd.in

```diff
@@ -40712,25 +40712,72 @@ ZoneHVAC:TerminalUnit:VariableRefrigerantFlow,
 
 \group Zone HVAC Radiative/Convective Units
 
+ZoneHVAC:Baseboard:RadiantConvective:Water:Design,
+       \min-fields 7
+  A1,  \field Name
+       \required-field
+       \type alpha
+       \reference BaseboardDesignObject
+  A2,  \field Heating Design Capacity Method
+       \type choice
+       \key HeatingDesignCapacity
+       \key CapacityPerFloorArea
+       \key FractionOfAutosizedHeatingCapacity
+       \default HeatingDesignCapacity
+       \note Enter the method used to determine the heating design capacity.
+       \note HeatingDesignCapacity = > selected when the design heating capacity value or autosize
+       \note is specified. CapacityPerFloorArea = > selected when the design heating capacity is
+       \note determine from user specified heating capacity per floor area and zone floor area.
+       \note FractionOfAutosizedHeatingCapacity = > is selected when the design heating capacity is
+       \note determined from a user specified fraction and the auto-sized design heating capacity.
+  N1, \field Heating Design Capacity Per Floor Area
+       \type real
+       \units W/m2
+       \minimum 0.0
+       \note Enter the heating design capacity per zone floor area.Required field when the heating design
+       \note capacity method field is CapacityPerFloorArea.
+  N2, \field Fraction of Autosized Heating Design Capacity
+       \type real
+       \minimum 0.0
+       \default 1.0
+       \note Enter the fraction of auto - sized heating design capacity.Required field when capacity the
```

Comment:
I see you just copied this note from the previous object, but if you can fix up the spacing it would be nice.  Change it to be just autosized, and then a  space after the period.  Then go through the other notes briefly and see if you spot any other weird spacing or punctuation.

Reply:
It's weird that this file/these lines do not show up as outdated.